ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  role_name: ocp4_workload_example
+  author: Red Hat GPTE, Wolfgang Kulhanek (wkulhane@redhat.com)
```

Comment:
Nope. I didn't write this. :-) Can't support it either... :-) role_name and description need updating, too, please.